core/domain/exp_jobs_one_off.py

```diff
@@ -902,3 +902,38 @@ def reduce(key, values):
             yield (key, len(values))
         else:
             yield (key, values)
+
+
+class RatioTermsAuditOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """Job that checks the number of ratio terms used by each state of an
+    exploration.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+
+        exploration = exp_fetchers.get_exploration_from_model(item)
+        for state_name, state in exploration.states.items():
+            interaction = state.interaction
+            exp_and_state_key = '%s %s' % (
+                item.id, state_name.encode('utf-8'))
+            if interaction.id == 'RatioExpressionInput':
+                if interaction.customization_args['numberOfTerms'].value > 10:
+                    yield (python_utils.UNICODE(
+                        interaction.customization_args['numberOfTerms'].value),
+                           exp_and_state_key)
```

Comment:
```suggestion
            if interaction.id == 'RatioExpressionInput':
                number_of_terms = (
                    interaction.customization_args['numberOfTerms'].value)
                if number_of_terms > 10:
                    yield (
                        python_utils.UNICODE(number_of_terms), exp_and_state_key
                    )
```

Reply:
This is not done yet?

Reply:
Actually, by mistake, I missed it. Updating the PR soon.